leakcanary-android/src/main/res/mipmap-anydpi-v26/leak_canary_icon_round.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
I don't know if this would work. If an OEM enables round icons and we give them an adaptive icon would the system automatically crop the background layer appropriately?

Reply:
I can't remember off-hand if this works. If you don't want any special behaviour for `roundIcon` then just don't specify one in the manifest.

Reply:
Hmm, interesting question.

I used the Android Studio "New Image Asset" tool to generate those files.

I think the use case you're describing cannot happen.
If they support adaptative icons, they can crop it however they want (and should not request round icon). If not, it means API < 26.

If they support round icons (API = 25), they'll get the round icons from the appropriate folder (not the adaptative one as it is declared in a `-v26` folder)

That file is there just so we can use the same name in the `roundIcon` field in the Manifest I guess.

Reply:
That's not quite correct. `roundIcon` still exists in API>26 - it's for apps which need to treat round icons specially. Think Chrome etc which want to do something differently when there's a circle mask (i.e. fill the circle with their icon rather than placing it on a background).

If you don't want any special behaviour for circles (and I don't think you do) then the simplest thing is to just remove the `roundIcon` attr from your manifest. The wizard has to include `roundIcon` for apps which still require it but for most users, it's no longer relevant.